massdriver.yaml

```diff
@@ -42,7 +42,75 @@ params:
     - cluster_networking
     - node_groups
     - core_services
+    - observability
   properties:
+    observability:
```

Comment:
If we like this structure we can make this a type and ref it from each cloud's cluster.

Reply:
I like that idea. I'll back it out to a type once I confirm it's working as expected in staging

Reply:
I would advise not extracting this to a separate type.  EKS will have log options the others won't (Cloudwatch, managed Opensearch), GKE same (Stackdriver) and same with AKS.

We should use the same structure, but not force them to use the same type.